projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad.php

```diff
@@ -812,6 +812,21 @@ function wpcom_log_launchpad_being_enabled_for_test_sites( $option, $value ) {
 	);
 }
 
+/**
+ * Returns whether or not the Launchpad is enabled for a given checklist.
+ *
+ * @param string $checklist_slug The checklist slug.
+ * @return bool True if the Launchpad is enabled, false otherwise.
+ */
+function wpcom_get_launchpad_task_list_is_enabled( $checklist_slug ) {
+	$is_enabled = wpcom_launchpad_checklists()->is_task_list_enabled( $checklist_slug );
+	if ( null !== $is_enabled ) {
+		return $is_enabled;
+	}
+
+	return wpcom_launchpad_checklists()->is_launchpad_enabled();
```

Comment:
Do you envision removing the fallback to `wpcom_launchpad_checklists()->is_launchpad_enabled()` here once we've refactored all the task lists to use `is_enabled_callback`? If so, maybe adding a note or `@todo` to that effect here would reduce confusion, because I can see us getting tripped up. They have similar names but map to two different values, if that makes sense.

Reply:
Yes, I should've made it clear. But I have also been wondering what "is enabled" means because if you look at the `is_launchpad_enabled` definition, we are also checking if the task list has incomplete tasks:
```php
public function is_launchpad_enabled() {
	$launchpad_screen = get_option( 'launchpad_screen' );
	if ( 'full' !== $launchpad_screen ) {
		return false;
	}

	return $this->has_active_tasks();
}
```
Maybe we should also have an `is_complete` state.

Reply:
Another thought: we could add a new callback that uses the `is_launchpad_enabled` method and add the `is_enabled_callback` to all existing task lists and make it mandatory on the definition. What do you think?

Reply:
> Another thought: we could add a new callback that uses the is_launchpad_enabled method and add the is_enabled_callback to all existing task lists and make it mandatory on the definition. What do you think?

Do you mean making `is_launchpad_enabled` the default callback for all task lists unless we specify otherwise? That works!

Reply:
I meant the same thing you said here https://github.com/Automattic/jetpack/pull/30913#discussion_r1205846448.  
I describe my idea here: https://github.com/Automattic/wp-calypso/issues/77407